### ISSUE TITLE
Add Automatic-Module-Name for JPMS

### ISSUE THREAD

**Issue by danielfernandez** [orphan-oss/ognl#52]
In order to have the OGNL jar artifact properly work as a dependency in JPMS (a _required_ module in a Java 9+ modular application), OGNL should choose a module name that gives dependent software some kind of guarantee on the module name that should be used for it, whatever the version. This can be done even if OGNL is not turned yet into a _module_ as such, by merely adding an `Automatic-Module-Name: <name>` entry to the `MANIFEST.MF` file.

This works as a kind of _reserve_ for module names and allows software using OGNL to properly express OGNL as a dependency in a `requires` clause in `module-info.java`, even if OGNL has not been modularised yet.

Note there are two main approaches for module naming (see [this](http://blog.joda.org/2017/04/java-se-9-jpms-module-naming.html)). Apache Commons OGNL has already chosen the reverse-DNS approach (see [their pom.xml](https://github.com/apache/commons-ognl/blob/51e64c812f71cab53368620ca934188eed6be4d6/pom.xml#L181)) so it is `org.apache.commons.ognl` there, but given OGNL 3 and OGNL 4 will be completely package- and code-incompatible (`ognl.*` vs `org.apache.commons.ognl.*` packages), I'm not sure it would make sense to adopt `org.apache.commons.ognl` here too as a module name for the sake of compatibility. So using simply `ognl` as name could be a sensible option here too. Especially given the fact that Apache Commons OGNL seems dead.

**Comment by lukaszlenart** [comment]
Yes, 👍 for a simple name `ognl`. And yes, Commons OGNL seems to be dead, but I still have hope that I will return to it someday :(

Or maybe just incorporate changes from Commons OGNL into OGNL4 🤔

**Comment by lukaszlenart** [comment]
Fixed with [6905a10](https://github.com/JCgH4164838Gh792C124B5/ognl/commit/6905a10ca2afc011e2c7da7418a5c0e2f55ded50)
